### ISSUE TITLE
initialRows still isn't working completely right

### ISSUE THREAD

**Issue by keathmilligan** [microsoft/terminal#6373]
# Environment

```none
Windows build number: 10.0.19041.0
Windows Terminal version (if applicable): 1.0.1401.0
```

# Steps to reproduce

Set InitialRows to 40.
Open Windows Terminal.
In Powershell, run:

```powershell
for ($i = 1; $i -le 40; $i++) { Write-Host; Write-Host -NoNewLine $i }; [console]::ReadKey()
```

# Expected behavior

You should see all 40 rows.

# Actual behavior

Instead, you only see rows 2-40. The area of the window that should contain the last row is not used:

![image](https://user-images.githubusercontent.com/2936304/83914040-132cd800-a736-11ea-918d-f2bda99111ad.png)

Now grab the bottom edge of the window, shrink it by one and then return it to its original size. Run the script above again and see that all 40 rows are now displayed with the last row being utilized.

![image](https://user-images.githubusercontent.com/2936304/83914091-2b9cf280-a736-11ea-8891-6e47320b9d95.png)

**Comment by DHowett** [comment]
Huh, I can't reproduce this.

* What's your DPI scaling?
* Can you run `mode con`?
* _Can you select the blank line under 40?_

**Comment by keathmilligan** [comment]
DPI scaling is 150% on a 4k display.

When I first open the window, before manually resizing, mode con shows 39 lines even though InitialRows is set to 40. And no, the blank line cannot be selected.

**Comment by DHowett** [comment]
Thanks! I thought we had a duplicate of this, but I think we thought we'd fixed it.

**Comment by zadjii-msft** [comment]
Hey I think we fixed this in 1.5 / 1.6 - there was #8527 (which has almost exactly the same title) which should be fixed

> 🎉This issue was addressed in #8584, which has now been successfully released as Windows Terminal Preview v1.6.10272.0.🎉

I'm more shocked that we didn't dupe that other issue to this one in the first place

/dup #8527
